pandas/tests/io/json/test_normalize.py

```diff
@@ -351,7 +353,7 @@ def test_json_normalize_errors(self):
                       errors='raise'
                       )
 
-    def test_nonetype_dropping(self):
+    def test_nonetype(self):
         # GH20030: Checks that None values are dropped in nested_to_record
```

Comment:
Can certainly delete this comment, though replace with a reference to the GitHub issue you are working on